spec/models/conference_spec.rb

```diff
@@ -1639,4 +1639,27 @@
       expect(free_ticket.price_cents).to eq(0)
     end
   end
+
+  describe 'after_update' do
+    let(:conference) { create(:full_conference) }
```

Comment:
then there is already a `conference` variable at the top: https://github.com/siddhantbajaj/osem/blob/007804f3ff4d43dd4ac2137d254cc6e1112839c2/spec/models/conference_spec.rb#L7

Although it is call subject, but we should use it here, so we don't have to define it twice. :wink: